addons/mail/static/src/thread/thread.xml

```diff
@@ -6,7 +6,7 @@
     <t t-set="isEmpty" t-value="thread.messages.length === 0"/>
     <div class="o-mail-thread overflow-auto pb-4 flex-grow-1 bg-view" t-ref="messages">
         <t t-if="!isEmpty">
-            <div class="d-flex flex-column" t-att-class="{'justify-content-end': !env.inChatter}" style="min-height:100%">
+            <div class="d-flex flex-column" t-att-class="{'justify-content-end': thread.type !== 'chatter' and thread.type !== 'mailbox'}" style="min-height:100%">
```

Comment:
"chatter" type (business documents) can be displayed inside thread of chat window too, which would break with current condition (being "in chatter" should come from which parent is actually used, not the type of the thread)

Reply:
I didn't know such a situation could occur. So, in that case, we want to justify the content at the end I guess? (both would make sense to me, to be honest)